### ISSUE TITLE
License Error 9.6.1

### ISSUE THREAD

**Issue by KatiaSishost** [mattermost/mattermost#26626]
I did a clean installation of version 9.6.0 and 9.6.1 when I started Trial... it told me that there was an error and it did not activate the Enterprise demo.

I have tried an old version 7.7.1 and this one activates the demo for me.

Mattermost Version: 9.6.0
Database Schema Version: 119
Build Number: 8083694714
Database: mysql

{
    "id": "api.license.add_license.invalid.app_error",
    "message": "Invalid license file.",
    "detailed_error": "",
    "request_id": "gt494qkj43d1fbxfdxmojhuibw",
    "status_code": 400
}

**Comment by amyblais** [comment]
Hi @KatiaSishost, I wasn't able to reproduce this just now. What error did you see, and can you help share detailed reproduction steps?

**Comment by amyblais** [comment]
I found out that there was a bug in recent versions where some test settings were added to the config that were meant for the cloud service testing. This will be fixed in v9.7.0.
 
To fix the issue on your end in the meanwhile, can you set the ``CloudSettings`` to the following and try again?

```
"CloudSettings": {
"CWSURL": "https://customers.mattermost.com",
"CWSAPIURL": "https://portal.internal.prod.cloud.mattermost.com",
"CWSMock": false
},
```

If this fails, you can also get a trial license by filling out the form on this website https://mattermost.com/trial/.
